web-server/v0.4/utils/datastore.js

```diff
@@ -0,0 +1,4 @@
+export default { 
+    elasticsearch: "http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.run.*/_search",
+    production: "http://pbench.perf.lab.eng.bos.redhat.com"
```

Comment:
Can you use "`pbench.example.com`"?

Reply:
Can you also just use hostnames, and pick the http scheme from the current we page?